ts/Series/Treegraph/TreegraphLayout.ts

```diff
@@ -222,19 +216,19 @@ extend(H.treeLayouts.Walker.prototype, {
             // If the node has children, perform the recursive first walk for
             // its children, and then calculate its shift in the apportion
             // function (most crucial part part of the algorythm).
-            let defaultAncestor = node.getLeftMostChild() as TreegraphNode;
+            let defaultAncestor = node.getLeftMostChild() as TreegraphNode.Node;
 
-            node.linksFrom.forEach(function (link): void {
-                treeLayout.firstWalk(link.toNode);
+            node.children.forEach(function (child): void {
+                treeLayout.firstWalk(child);
                 defaultAncestor = treeLayout.apportion(
-                    link.toNode,
+                    child,
                     defaultAncestor
                 );
             });
             treeLayout.executeShifts(node);
 
-            let leftChild = node.getLeftMostChild() as TreegraphNode;
-            let rightChild = node.getRightMostChild() as TreegraphNode;
+            let leftChild = node.getLeftMostChild() as TreegraphNode.Node;
```

Comment:
Do we still need this cast as `TreegraphNode.Node`?

Reply:
yes